recipe/meta.yaml

```diff
@@ -18,10 +18,12 @@ source:
     # https://github.com/conda-forge/staged-recipes/pull/199
     #
     - test_Makefile.in.patch
+    # Disable shared Fortran API building on OSX
+    - osx_configure.patch  # [osx]
 
 build:
-  number: 10
-  skip: True  # [py==36]
+  number: 11
+  skip: True  # [win and py==36]
```

Comment:
Hmm...should be added. Raised issue ( https://github.com/conda/conda-docs/issues/443 ) to get `py36` added.